ext/js/language/language-transforms.js

```diff
@@ -52,3 +52,21 @@ export function prefixInflection(inflectedPrefix, deinflectedPrefix, conditionsI
         conditionsOut
     };
 }
+
+/**
+ * @param {string} inflectedLiteral
+ * @param {string} deinflectedLiteral
+ * @param {string[]} conditionsIn
+ * @param {string[]} conditionsOut
+ * @returns {import('language-transformer').Rule}
+ */
+export function literalInflection(inflectedLiteral, deinflectedLiteral, conditionsIn, conditionsOut) {
```

Comment:
back on Yezichak I called these wholeWordInflection I think. Maybe smth like that is easier to understand for folks who will be editing these language files